ros/src/tl_detector/helpers.py

```diff
@@ -0,0 +1,13 @@
+import tensorflow as tf
```

Comment:
This file is no longer exists in `master`. I suggest just move `eval.ipynb` and `environment.yaml` to a new branch, started from the current `master` branch. It seems like your branch is merged with the old SSD Classifier branch that is not currently in master.